apps/prairielearn/python/prairielearn/html_utils.py

```diff
@@ -337,5 +365,9 @@ def inner_html(element: lxml.html.HtmlElement) -> str:
 
 
 def escape_invalid_string(string: str) -> str:
-    """Wrap and escape string in `<code>` tags."""
+    """Wrap and escape string in `<code>` tags.
+
+    Returns:
+        The sanitized user input wrapped in a code block.
```

Comment:
```suggestion
        A string representing the sanitized user input wrapped in a code block.
```

Reply:
I think the type of the return will be included so maybe not needed.